bundles/CoreBundle/Resources/config/pimcore/default.yaml

```
@@ -293,6 +293,14 @@ pimcore:
               'abbr', 'option', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6'
           ]
 
+    templating_engine:
+        twig:
+            security_policy:
```

```suggestion
            sandbox_security_policy:
```
Maybe the naming should be a bit more specific here, otherwise someone could assume that this applies globally?

---

done